Gemfile.lock

```diff
@@ -1053,4 +1053,4 @@ RUBY VERSION
    ruby 3.3.1p55
```

Comment:
If it ran with the new version, would it also change this?

Reply:
Uh ... I think it would not (by default anyway) - the default behavior is to read the version from the file, compare to whatever's running, and if they are different, first download/install the specified version from the file, then resume/repeat the install process (maybe there is a setting for this I'm not aware of, but that's the default logic).

There are explicit options (`bundle update --ruby` and `bundle update --bundler`) which bump those deps in the lockfile.